src/devices/LidarLiteV3/SystemStatus.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.DistanceSensor.Models.LidarLiteV3
+{
+    /// <summary>
+    /// System status flags
+    /// </summary>
+    [Flags]
+    public enum SystemStatus
+    {
+        /// <summary>
+        /// System error detected during measurement
+        /// </summary>
+        ProcessError = 0x40,
+
+        /// <summary>
+        /// Health status, indicating reference and receiver bias are operational
+        /// </summary>
+        Health = 0x20,
```

Comment:
Should this be "Healthy"?

Reply:
These are the direct name of the flags from the device documentation. Not saying we shouldn't change it, but one of the design idea I've been thinking about is to expose a higher-level API (different class, possibly different namespace) for this device, and while also maintaining a bare metal low level version that maps one-to-one to the device.  These flags would remain the same and kept in the low level API.

So I see two classes of users:

Typical User --> High-Level API --> Low-Level API --> Device
Advanced User --> Low-Level API --> Device

Thoughts?